### PR TITLE
Responsive Design - Milestone 1

### PR DESCRIPTION
**Pull request by MaiRizk**
The changes in this PR;

1- Structure the header, footer, highlights, and latest funds sections using semantic html.
2- Styling all the sections using css
3- Make responsive design for mobile, tablet, and desktop for all sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    <a href="#" id="topData">
+        <span>Calling young digital talent in the Netherlands – Be recognized in the T500 →</span>
+    </a>
+</header>
+<!-- END Header -->
+
+<!-- Start Highlights -->
+<section class="first-section">
+    <div class="grid-container grid">
+        <div class="first-grid highlights">
```
Comment (henatan99) [inline index.html]: Better to use article tag here instead of div. Apply same changes for the other highlights article elements

File: index.html [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fas fa-envelope"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="footer-links flex">
```
Comment (henatan99) [inline index.html]: Better to change this div tag to nav-tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+            </div>
+        </div>
+    </div>
+</section>
+<!-- END latest founding -->
+
+<!-- START footer -->
+<footer class="flex-column">
+    <section class="top-footer flex-column flex">
+        <div class="socialmedia-icons flex">
```
Comment (henatan99) [inline index.html]: Better change this div tag to nav tag

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes required :recycle: 
Hi there, what an awesome work :+1: :clap: :clap: . You did really great, but there are only a few mandatory changes you need to make before your project gets approved. 

### Reminder: This is your first review and you will be left with a maximum of two more reviews. If your project doesn't get approved at the third review you will be asked to repeat the project. Please make sure you make all the requested changes before you submit for re-review. 

## To Highlight 
- Your page looks very close to the original page. Business requirement is nicely met 
- Clean and DRY code 
- Only missed very few mandatory requirements 
-
## To Improve [MANDATORY] 
- All fontawsome icons are not displaying. I recommend you put the fontawsome CDN link at the bottom of the body.
- Use article tag for highlight section elements. See inline comment 
- Wrap social-media-icons and footer-links menu elements (ul) in  nav tag in the footer section

### Please work on the requested changes and submit for re-review.

Should you have any queries, you can get me via:
[LinkedIn](https://www.linkedin.com/in/henok-mekonnen-2a251613/) | [Twitter](https://twitter.com/henatan99) | Henok Mekonnen on Slack

File: index.html (hunk 1 above)
Comment (IbrohimRasulov) [reply index.html]: Thank you for the review and suggestions. We corrected the parts you mentioned.

File: index.html (hunk 2 above)
Comment (IbrohimRasulov) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (IbrohimRasulov) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (IbrohimRasulov) [reply index.html]: We checked the font awesome icons, and they are functioning normally.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @MaiRizk & @IbrohimRasulov

Good job on implementing all the suggested changes. The website is perfectly cloned. Well done. 🥳🥳🥳

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**

Cheers and Happy coding!👏👏👏

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Pull request merged**